packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -42,6 +42,7 @@
     "cross-env": "5.2.0",
     "esbuild": "^0.19.5",
     "eslint": "8.52.0",
+    "esbuild-plugin-pino": "^2.1.0",
```

Comment:
esbuild-plugin-pino is required in order to bundle core-bundle due to the existence of pino-logger